repository-ydb-v2/src/main/java/tech/ydb/yoj/repository/ydb/table/YdbTable.java

```diff
@@ -67,9 +67,12 @@
 public class YdbTable<T extends Entity<T>> implements Table<T> {
     @Getter
     private final Class<T> type;
+
+    @Getter
```

Comment:
If I remember correctly, we've been gradually trying to get rid of Lombok? Maybe we should write this manually in new code?

Reply:
Hmmm, I don't remember that we wanted to get rid of Lombok :hot_pepper:
And using `@Getter` keeps the same code style as the rest of the class.